java/api/src/main/java/io/ray/api/call/ActorCreator.java

```diff
@@ -60,4 +60,10 @@ public ActorCreator<A> setRuntimeEnv(RuntimeEnv runtimeEnv) {
     builder.setRuntimeEnv(runtimeEnv);
     return this;
   }
+
+  /** Set the namespace that this actor uses. */
+  public ActorCreator<A> setNamespace(String namespace) {
```

Comment:
Should the namespace always set along with a name? 
If yes, we should change the API to `setName(name, namespace)`
The semantic seems unclear if only setting namespace without name. Not Sure if Python allows this.
@simon-mo @jovany-wang

Reply:
yes namespace is pretty coupled with name